### ISSUE TITLE
Shoud check for ascending series should apply to stop_times of the interval?

### ISSUE THREAD

**Issue by h-mayorquin** [NeurodataWithoutBorders/nwbinspector#375]
I have the following situation:

```
sleep_states = nwbfile.processing["behavior"]["SleepStates"]
sleep_states.to_dataframe().head(8)
```

Which outputs the following:

| id | start_time | stop_time | label                |
|----|------------|-----------|----------------------|
|  0 |          1 |         1 | WAKEtheta_ThDt       |
|  1 |          1 |         3 | QWake_ThDt           |
|  2 |          1 |         3 | QWake_noRipples_ThDt |
|  3 |          1 |        50 | WAKEnontheta         |
|  4 |          1 |       100 | WAKEstate            |
|  5 |          3 |         4 | WAKEtheta_ThDt       |
|  6 |          5 |        13 | QWake_ThDt           |
|  7 |          5 |        13 | QWake_noRipples_ThDt |

And it is flagged by the inspector with the following warning:
```
0.1  fCamk1_200827_sess9.nwb: check_time_interval_time_columns - 'TimeIntervals' object with name 'SleepStates'
       Message: ['stop_time'] are time columns but the values are not in ascending order. All times should be in seconds with respect to the session start time.
```

Looking at the check it seems that it assumes that all time columns should be ascending which does not seem to be true for my case. 

https://github.com/NeurodataWithoutBorders/nwbinspector/blob/2ddada2cf7481127d35616ece0e6e7d17e332d29/src/nwbinspector/checks/tables.py#L47-L71

Am I missing something? What do you think?

**Comment by bendichter** [comment]
Hmm yes I guess if you have overlapping intervals it might be impossible for start and stop time to both be ascending. That appears to be the case here. What importance level is this in the inspector?

**Comment by h-mayorquin** [comment]
```

0  BEST_PRACTICE_VIOLATION
==========================
...
0.1  fCamk1_200827_sess9.nwb: check_time_interval_time_columns - 'TimeIntervals' object with name 'SleepStates'
       Message: ['stop_time'] are time columns but the values are not in ascending order. All times should be in seconds with respect to the session start time.
...

````

**Comment by bendichter** [comment]
OK, maybe we should disable this for `stop_time`?

**Comment by h-mayorquin** [comment]
Yes, that makes sense to me. What about other columns?

**Comment by CodyCBakerPhD** [comment]
Very interesting

Solution would indeed just be to skip the `stop_time` column for that check

Past assumption from what we've usually seen was simply a sequence of disjoint intervals, so check wants to make sure those intervals are in order according to when each intervals started
